packages/intl/src/strings.ts

```diff
@@ -1982,6 +1982,9 @@ All attachments will be downloaded & cached again on access.
       ]()}`
     }),
   backupReadyToDownload: () => t`Your backup is ready to download`,
+  vaultLocked: () => t`Vault locked`,
+  vaultLockedSuccessfully: () => t`Vault locked successfully`,
```

Comment:
This is unnecessary. The word "successfully" is not meaningful. We can just use `vaultLocked` instead.